Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -168,6 +174,49 @@ class SearchSettingsTableViewController: ThemedTableViewController, FeatureFlagg
             cell.editingAccessoryView = toggle
             cell.selectionStyle = .none
             cell.accessibilityIdentifier = AccessibilityIdentifiers.Settings.Search.disableSearchSuggestsInPrivateMode
+        case .firefoxSuggestSettings:
+            switch indexPath.item {
+            case ItemSuggestionNonSponsored:
+                let setting = BoolSetting(
+                    prefs: profile.prefs,
+                    theme: themeManager.currentTheme,
+                    prefKey: PrefsKeys.FirefoxSuggestShowNonSponsoredSuggestions,
+                    defaultValue: profile.prefs.boolForKey(PrefsKeys.FirefoxSuggestShowNonSponsoredSuggestions) ?? true,
+                    titleText: String.localizedStringWithFormat(.Settings.Search.Suggest.ShowNonSponsoredSuggestionsTitle,
+                                                                AppName.shortName.rawValue),
+                    statusText: String.localizedStringWithFormat(.Settings.Search.Suggest.ShowNonSponsoredSuggestionsDescription,
+                                                                 AppName.shortName.rawValue)
+                )
+                setting.onConfigureCell(cell, theme: themeManager.currentTheme)
+                setting.control.addTarget(self, action: #selector(didToggleEnableNonSponsoredSuggestions), for: .valueChanged)
+                cell.editingAccessoryView = setting.control
+                cell.selectionStyle = .none
+            case ItemSuggestionSponsored:
+                let setting = BoolSetting(
+                    prefs: profile.prefs,
+                    theme: themeManager.currentTheme,
+                    prefKey: PrefsKeys.FirefoxSuggestShowSponsoredSuggestions,
+                    defaultValue: profile.prefs.boolForKey(PrefsKeys.FirefoxSuggestShowSponsoredSuggestions) ?? true,
+                    titleText: .Settings.Search.Suggest.ShowSponsoredSuggestionsTitle,
+                    statusText: String.localizedStringWithFormat(.Settings.Search.Suggest.ShowSponsoredSuggestionsDescription,
+                                                                 AppName.shortName.rawValue)
+                )
+                setting.onConfigureCell(cell, theme: themeManager.currentTheme)
+                setting.control.addTarget(self, action: #selector(didToggleEnableSponsoredSuggestions), for: .valueChanged)
+                cell.editingAccessoryView = setting.control
+                cell.selectionStyle = .none
+            case ItemSuggestionLearn:
+                cell.accessibilityLabel = String.localizedStringWithFormat(.Settings.Search.AccessibilityLabels.LearnAboutSuggestions,
```

Comment:
This is _definitely_ out of my wheelhouse, but I'm wondering, why does this cell need an explicit accessibility label, but the `BoolSetting`s (and some of the other text cells?) don't?

/cc @adudenamedruby

Reply:
I can't speak for why @tiftran did it like this.

Theoretically, the accessibility label is what's read out by VO, and defaults to the text in the cell, if the a11y label isn't set.

Looking at the strings closely, it looks like these strings are the same so..... but it's good practice to add them, though.

We're still sorting out our accessibility stuff, which is why I didn't make an explicit thing here.

Reply:
🙈  sorry i was trying to follow convention I saw that `BoolSetting` didn't specify an accessibility label. and text cells did?